frontend/src/app/features/assistance-selection/assistance-selection.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component } from '@angular/core';
+import config from "../../core/constants/config.json"
+import { AssistanceLevel } from '../../core/interfaces/assistance-level';
+
+@Component({
+  selector: 'assistance-selection',
+  templateUrl: './assistance-selection.component.html',
+  styleUrls: ['./assistance-selection.component.scss']
+})
+export class AssistanceSelectionComponent {
+  levels: AssistanceLevel[] = config['assistance-levels']
+  selectedLevel: string = ''
+
+  handleSelectLevel= (level: AssistanceLevel) : void => {
+    this.selectedLevel = level.key
+  }
+
+  // isSelected = (level: AssistanceLevel) : boolean => {
```

Comment:
Since this code has been committed to the repository at some point, there's no need to have it here, commented out.